framework/helpers/BaseStringHelper.php

```diff
@@ -305,4 +305,30 @@ public static function normalizeNumber($value)
 
         return $value;
     }
+
+    /**
+     * Returns base64-encoded string for $input string with url safe alphabet.
+     * It refers to the following RFC chapter https://tools.ietf.org/html/rfc4648#page-7
```

Comment:
Suggest instead:

     * Implements the "Base 64 Encoding with URL and Filename Safe Alphabet" of RFC 4648